cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -135,13 +135,20 @@ func WriteKubeletConfigFiles(cfg *kubeadmapi.InitConfiguration, kubeletConfigDir
 		_, err := os.Stat(filepath.Join(kubeletDir, kubeadmconstants.KubeletInstanceConfigurationFileName))
 		if os.IsNotExist(err) {
```

Comment:
this code is a bit messy but can be removed once the feature gate is removed.
if the instance file is missing, maybe we can just cause and error.